src/main/java/org/chameleoncloud/ChameleonProject.java

```diff
@@ -17,7 +15,10 @@ public ChameleonProject(final String id, final String nickname) {
     }
 
     @Override
-    public int compareTo(@NotNull ChameleonProject o) {
+    public int compareTo(ChameleonProject o) {
+        if (o == null) {
```

Comment:
Not worth bringing in a dependency for the NotNull annotation, so a simple null check makes more sense.